examples/nextjs/README.md

```diff
@@ -0,0 +1,5 @@
+## Next.js App Router Course - Starter
```

Comment:
Let's update the README just a little bit so that it includes some context about Corbado.